doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -0,0 +1,90 @@
+.. _thread_ot_memory:
+
+OpenThread memory requirements
+##############################
+
+This page provides information about the layout and the amount of Flash and RAM memory required by the sample using the OpenThread stack.
+These values are useful to see whether your application has enough space for running on paricular platforms in different OpenThread configurations.
+
+
+Flash and RAM requirements for nRF52840 and nRF52833
+****************************************************
+
+The following tables list flash memory and RAM requirements for the provided samples built using the OpenThread libraries.
+The values change depending on the application and hardware platform.
+
+The values in the tables are provided for the CLI example, which works with all OpenThread calls.
+For this reason, these values are the best approximation of the memory requirements.
+
+Samples were compiled with the default :file: `prj.conf`, MTD variant of the CoAP Client has been compiled with the :file: `overlay-mtd.conf`.
```

Comment:
```suggestion
Samples were compiled with the default :file:`prj.conf`, MTD variant of the CoAP Client has been compiled with the :file:`overlay-mtd.conf`.
```
Maybe an introductory section "How the values were compiled" would be good for information in lines 13-32?
You could also explain in such section why the samples were compiles with these variants.